### PR TITLE
Use CMC api v2

### PR DESCRIPTION
**Pull request by LucoLe**
#### Summary

Changes the configs so v2 of CMC API is used.

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

**Comment by LucoLe** [comment]
I'm not sure that I understand the full extent of this task. Also, I exceed the rate limit when I test this locally.

File: config/test.exs [hunk 1]
```diff
@@ -33,7 +33,7 @@ config :sanbase, Sanbase.Timescaledb, blockchain_schema: nil
 
 config :sanbase, Sanbase.Auth.Hmac, secret_key: "Non_empty_key_used_in_tests_only"
 
-config :sanbase, Sanbase.ExternalServices.Coinmarketcap, sync_enabled: false
+config :sanbase, Sanbase.ExternalServices.Coinmarketcap2, sync_enabled: false
```
Comment (tspenov) [inline config/test.exs]: I think this also uses the `v1` version - https://github.com/santiment/sanbase2/blob/6d6e56b54248f1502cd8fc413f3398ac7c5065fc/lib/sanbase/external_services/coinmarketcap2/ticker.ex#L40
I think the idea is to migrate to `v2` -> https://api.coinmarketcap.com/v2/ticker/ ... see here: https://coinmarketcap.com/api/

**Comment by IvanIvanoff** [comment]
As it was said, this should migrate the API to v2. Change the used url to  https://api.coinmarketcap.com/v2/ticker/ and run it. It will become obvious what and where should be changed as you'll run into parsing issues and not-found keys issues.

**Comment by bobbyong** [comment]
Hi guys have you tried our API at http://coingecko.com/api

It's public, open, and freely available. Its quite easy to consume with little to no worry about limits.
It's already been used by many good crypto projects. Happy to answer any questions or concerns you may have :)

**Pull request closed without merging**